less/angular/pages/goggles.less

```diff
@@ -0,0 +1,189 @@
+.goggles-backgrounded {
+  position: relative;
+  z-index: 1;
+  background: inherit;
+  overflow: hidden;
+
+  &.goggles-xray-image {
+    padding-bottom: 3em;
+
+    img {
+      position: absolute;
+      z-index: 0;
+      padding: 0;
+    }
+  }
+
+  .goggles-header {
+    display: flex;
+    flex-direction: row;
+
+    position: relative;
+    background: white;
+    width: 100%;
+    padding: 3em 0 2em;
+
+    h1 {
+      flex: auto;
+      align-self: stretch;
+      margin: 0;
+    }
+
+    .goggles-bookmark {
+      flex: 0 1 auto;
+      align-self: stretch;
+      text-align: right;
+
+      p {
+        text-align: center;
+      }
+    }
+
+    .goggles-bookmark-link {
+      color: black;
+      text-decoration: none;
+      font-weight: bold;
+
+      &:hover, &:active, &:visited {
+        color: black;
+        text-decoration: none;
+      }
+
+      span {
+        display: inline-block;
+        border: none;
+        background: #FFBE23;
+        height: 2em;
+        vertical-align: middle;
+        font-size: 120%;
+        line-height: 2em;
+        padding: 0 0.75em;
+
+        &:first-child {
+          border-right: 2px solid grey;
+          border-radius: 0.552em 0 0 0.552em;
+        }
+
+        &:last-child {
+          border-radius: 0 0.552em 0.552em 0;
+        }
+      }
+    }
+  }
+
+  .goggles-box {
+    position: relative;
+    z-index: 1;
+  }
+
+  .goggles-main-box {
+    background: white;
+    width: 30%;
+    min-width: 15em;
+    margin: 1em auto;
+    padding: 1em;
+
+    h1 {
+      margin: 0.25em 0 1em;
+      text-align: center;
+    }
+
+    .goggles-activate-button {
+      border: none;
+      background: #FFBE23;
+      border-radius: 1em;
+      padding: 0.4em 1em;
+    }
```

Comment:
try adding margin-top:-1rem; for nice vertical alignment. see if it breaks anywhere.

style button as currently on homepage
![screenshot 2015-01-29 15 35 44](https://cloud.githubusercontent.com/assets/53411/5965754/97a53f7e-a7cc-11e4-8e9b-a5da299d8f4a.png)